examples/consumer-driven-contracts/payment-producer/pom.xml

```diff
@@ -15,14 +15,14 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-dependencies</artifactId>
-            <version>${spring.boot.version}</version>
+            <version>2.2.0.RELEASE</version>
```

Comment:
since you changed the parent pom this is not needed right ?

Reply:
I reverted the changes i made in parent pom and its using older version of spring boot.

Reply:
I mean the changes in this pom: https://github.com/intuit/karate/blob/develop/examples/consumer-driven-contracts/pom.xml#L19 - but it is ok I will do it